astro/src/content/docs/get-started/core-concepts/tenants.mdx

```diff
@@ -120,29 +120,7 @@ Connectors can be enabled on a per tenant basis with a Connector policy.
 
 <img src="/img/docs/get-started/core-concepts/tenant-connector-tab.png" alt="The Tenant Connector policy configuration tab." width="1200" role="shadowed" />
 
-Full documentation on [Connectors](/docs/lifecycle/migrate-users/connectors/) and [Connector Policies](/docs/lifecycle/migrate-users/connectors/) can be found [here](/docs/lifecycle/migrate-users/connectors/).
-
-## Add Connector Policy Dialog
-
-If you click on the `Add policy` button on this page you will be presented with the following dialog.
-
-<img src="/img/docs/get-started/core-concepts/tenant-configuration-connector-add-policy.png" alt="Add Connector Policy" width="300" role="box-shadow" />
-
-### Form Fields
-
-<APIBlock>
-  <APIField name="Connector" required>
-    The Connector to be used for this policy.
-  </APIField>
-  <APIField name="Domains" optional defaults='["*"]'>
-    One or more line separated domains to be used to filter incoming authentication requests. To match all incoming email addresses, a single entry using an asterisk `*` can be used.
-  </APIField>
-  <APIField name="Migrate user" optional>
-    When selected, migrate the user from the Connector into FusionAuth so that future authentications will use FusionAuth and not the Connector.
-  </APIField>
-</APIBlock>
-
-<br/>
+You can find details on adding a connector [below](#add-connector-policy-dialog). Full documentation on Connectors and Connector Policies can be found [here](/docs/lifecycle/migrate-users/connectors/).
```

Comment:
I haven't used a ScrollRef component anywhere. I'm not sure if we are supposed to use one or the other, but I see 100+ cases of this style to link to a heading somewhere else on the page.

I updated this because the ToC was weird for this page. `Tenant Configuration` goes through each of the tabs and documents them, but the way the headings were laid out meant that ToC shows `Add Connector Policy Dialog` then the page content continues with more tabs. It was like "we interrupt your regularly scheduled form field docs to bring you...this other dialog."

Reply:
I love the change, makes total sense and I appreciate the ownership of making docs better even though it wasn't strictly in the scope of change!

I don't know why we use scrollref sometimes. Asked: https://inversoft.slack.com/archives/C02H35R1Q6N/p1713815665458489

Reply:
For future reference using the markdown syntax is preferred over the `ScrollRef` component. The component was part of an abandoned smooth scroll something or another.